src/pages/project/new_project/CreateProject.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { PageHeader } from '../../../components/PageHeader';
+import { StyledPageContent } from '../../../components/styled/Wrappers';
+import EmptyProjectForm from './EmptyProjectForm';
+import NFRProject from './NFRProject';
+
+const CreateProject = () => {
+  const { t } = useTranslation();
+
+  return (
+    <StyledPageContent>
+      <PageHeader>{t('project.create_project')}</PageHeader>
+      <Box
+        sx={{
+          maxWidth: '55rem',
+          display: 'flex',
+          flexDirection: 'column',
+          gap: '2rem',
+        }}>
+        <NFRProject />
+        <EmptyProjectForm />
+      </Box>
+    </StyledPageContent>
+  );
+};
+
+export default CreateProject;
```

Comment:
Personlig liker jeg best å bruke named export (`export const CreateProject`), da jeg syns det ofte forenkler renaming av ting. Om komponenten her renames til feks `CreateProjectForm` i fremtiden, vil default export gjøre at den fortsatt vil kalles `CreateProject` andre plasser, helt til vi manuelt går inn å endrer alle plasser det trengs. Med named exports blir man tvunget til å bruke samme navn overalt

Men det er på ingen måte feil, som om du liker default exports bedre er det helt fair!
(gjelder alle komponentene med default export)

Reply:
Ja, gode poeng. Jeg gjør som du foreslår bortsett fra i de to som skal ha lazy import, de får beholde samme struktur som de andre med lazy import.